lib/helpers.js

```diff
@@ -132,10 +132,13 @@ function getToken(userInfo) {
       if (profile) {
         envvars.AWS_PROFILE = profile.value;
       }
+      for (const envvar of env) {
+        envvars[envvar.name] = envvar.value ?? ""
```

Comment:
why not `envvar.value || ""`?

Also, the `if(profile)` above is not needed anymore, right?